modules/mod_login/Helper/LoginHelper.php

```diff
@@ -63,4 +63,33 @@ public static function getType()
 
 		return (!$user->get('guest')) ? 'logout' : 'login';
 	}
+
+	/**
+	 * Retrieve the URL for the registration page
+	 *
+	 * @return  string
+	 */
+	public static function getRegistrationUrl($params)
+	{
+		$regLink = 'index.php?option=com_users&view=registration';
+		$regLinkMenuId = $params->get('customRegLinkMenu', 0);
+
+		// If there is a custom menu item set for registration => override default
+		if ($regLinkMenuId !== '' && $regLinkMenuId !== '0')
```

Comment:
This can be simplified.
```suggestion
		if ($regLinkMenuId)
```